### PR TITLE
consume APIs

### PR DESCRIPTION
**Pull request by sameh080081**
this branch is to learn how to consume API with post & get

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,49 @@
...
+const list = document.querySelector('.list');
+
+function display(arr) {
+  list.innerHTML = '';
+  arr.forEach((element) => {
+    list.innerHTML += `
+      <li>${element.user} : ${element.score}</li>
+       `;
+  });
+}
```
Comment (DammyShittu) [inline src/index.js]: - [ ] According to ES6 syntax you are required to use arrow functions. Kindly refactor this function.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,49 @@
...
+
+async function getScores(url) {
+  const response = await fetch(url);
+  const data = await response.json();
+  console.log(data.result);
```
Comment (DammyShittu) [inline src/index.js]: - [ ] Kindly remove this console log as it is unnecessary.

File: src/index.js [hunk 3]
```diff
@@ -1 +1,49 @@
...
+  display(data.result);
+}
```
Comment (DammyShittu) [inline src/index.js]: - [ ] Refactor to arrow function.

File: src/index.js [hunk 4]
```diff
@@ -1 +1,49 @@
...
+  })
+    .then((res) => res.json())
+    .then((data) => console.log(data));
+  nameInput.value = '';
+  scoreInput.value = '';
+}
+
+submit.addEventListener('click', () => {
+  addScore();
+});
```
Comment (DammyShittu) [inline src/index.js]: - [ ] Kindly refactor to arrow function.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @sameh080081,

## Highlights
✔️ Descriptive PR
✔️ Leaderboard score is saved

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @sameh080081, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**